src/controller/python/chip/clusters/Attribute.py

```diff
@@ -554,14 +554,14 @@ def _handleAttributeData(self, path: AttributePathWithListIndex, status: int, da
                 tlvData = chip.tlv.TLVReader(data).get().get("Any", {})
                 attributeValue = tlvData
 
-            self._cache.UpdateTLV(path, attributeValue)
+            self._cache.UpdateTLV(path, version, attributeValue)
```

Comment:
We're not doing much with this version, and not actually exposing this to users to see and interact with. Please file an issue to do so.

Reply:
https://github.com/project-chip/connectedhomeip/issues/14587